benchmarks/tpcds/load_postgres.py

```
@@ -0,0 +1,79 @@
+        f"port={options.pg_port}"
+    )
+
+
+def _load_table(connection, table, schema):
+    """Copy one base table from the DuckDB file into PostgreSQL, replacing it."""
+    target = f'pg.{schema}."{table}"'
+    source = f'src.main."{table}"'
+    connection.execute(f"DROP TABLE IF EXISTS {target}")
+    # CREATE ... AS SELECT streams the DuckDB table straight into PostgreSQL
+    # through the attached connection, so both sources hold identical rows.
+    connection.execute(f"CREATE TABLE {target} AS SELECT * FROM {source}")
```

Ruff (0.15.20)

[error] Possible SQL injection vector through string-based query construction

(S608)